docs/KustoSource.md

```diff
@@ -100,6 +100,19 @@ All the options that can be used in the Kusto Source can be found in KustoSource
 Options are - 'ForceSingleMode', 'ForceDistributedMode'.
 Scala and Java users may take these options from com.microsoft.kusto.spark.datasource.ReadMode.
 
+  The following is the behavior of the connector in either of these read modes
+
+  * **ForceSingleMode** : In force single mode Direct query and get the results
+
+  * **ForceDistributedMode** : When you hit Kusto [query limits](https://aka.ms/kustoquerylimits) (memory / number of records), data is [exported](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/management/data-export/export-data-to-storage) to Blob storage (export containers) , then the exported data is read and processed into a Dataframe. 
+
+  * **Automatic selection of the mode** : In case these options are not specified, the connector tries to count the approximate row counts and the approximate time the query takes. If the approximate row counts exceed the query limits specified above (or) the query times out (or) a memory limit exception is hit from the query, the connector will automatically use **ForceDistributedMode**.
```

Comment:
Just a suggestion, i dont think we need to share the whole logic ...
```suggestion
  * **Default selection of the mode** : In case these options are not specified, the connector tries to approximate row count and the time the query would take. If the approximated row count exceeds the query limits specified above, the connector will automatically use Distributed mode.
```